test/rng_testsuite/extra_methods_tests/distributions_methods.pass.cpp

```diff
@@ -0,0 +1,269 @@
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<float, 8>>>();
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<float, 16>>>();
+    if (err)
+    {
+        std::cout << "Test FAILED" << std::endl;
+    }
+    global_err += err;
+    err = 0;
+
+    std::cout << "---------------------------------------------------" << std::endl;
+    std::cout << "uniform_real_distribution<double>" << std::endl;
+    std::cout << "---------------------------------------------------" << std::endl;
+    err += test<oneapi::dpl::uniform_real_distribution<double>>();
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<double, 1>>>();
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<double, 2>>>();
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<double, 3>>>();
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<double, 4>>>();
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<double, 8>>>();
+    err += test<oneapi::dpl::uniform_real_distribution<sycl::vec<double, 16>>>();
+    if (err)
+    {
+        std::cout << "Test FAILED" << std::endl;
+    }
+    global_err += err;
+    err = 0;
+
+    std::cout << "---------------------------------------------------" << std::endl;
+    std::cout << "normal_distribution<float>" << std::endl;
+    std::cout << "---------------------------------------------------" << std::endl;
+    err += test<oneapi::dpl::normal_distribution<float>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<float, 1>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<float, 2>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<float, 3>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<float, 4>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<float, 8>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<float, 16>>>();
+    if (err)
+    {
+        std::cout << "Test FAILED" << std::endl;
+    }
+    global_err += err;
+    err = 0;
+
+    std::cout << "---------------------------------------------------" << std::endl;
+    std::cout << "normal_distribution<double>" << std::endl;
+    std::cout << "---------------------------------------------------" << std::endl;
+    err += test<oneapi::dpl::normal_distribution<double>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<double, 1>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<double, 2>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<double, 3>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<double, 4>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<double, 8>>>();
+    err += test<oneapi::dpl::normal_distribution<sycl::vec<double, 16>>>();
+    if (err)
+    {
+        std::cout << "Test FAILED" << std::endl;
+    }
+    global_err += err;
+    err = 0;
+
+    if (global_err)
+    {
+        return 1;

```

Comment:
Can it be combined somehow with `TestUtils::done` below?

Reply:
I suppose no, because TestUtils::done prints that test was skipped in case the argument != 1 (which will be not true for our case)

Reply:
ok. Then it can be done via `EXPECT_TRUE`. Will it work for your case?

Reply:
This check was unnecessary, I also changed checks for other RNG tests